src/lib/schema.ts

```diff
@@ -2,7 +2,7 @@ import { join } from 'path';
 import { makeExecutableSchema } from '@graphql-tools/schema';
 import { loadFilesSync } from '@graphql-tools/load-files';
 import { mergeTypeDefs } from '@graphql-tools/merge';
-import graphQLLetConfig from '../.graphql-let.yml';
+import graphQLLetConfig from '../../.graphql-let.yml';
```

Comment:
I think that the files in the `src` folder shouldn't get any information from the `/ root` folder. How about putting the config file into the `src` folder?

Reply:
I can't. I CANT. I CANT!!!

Reply:
```
[ graphql-let ] ENOENT: no such file or directory, open '/Users/wdever/Documents/Projects/covast-frontend-web/.graphql-l
et.yml'
error - ./src/lib/viewer.graphql
Error: ENOENT: no such file or directory, open '/Users/wdever/Documents/Projects/covast-frontend-web/.graphql-let.yml'
```

Bcuz of this error. I can't move the `graphql-let.yml` to `src` directory. Next couldn't know the location of `graphql-let.yml`